elixir/config/runtime.exs

```diff
@@ -233,4 +233,37 @@ if config_env() == :prod do
   config :workos, WorkOS.Client,
     api_key: compile_config!(:workos_api_key),
     client_id: compile_config!(:workos_client_id)
+
+  # Sentry
+
+  api_external_url_host = URI.parse(compile_config!(:api_external_url)).host
+
+  sentry_environment_name =
+    case api_external_url_host do
+      "api.firezone.dev" -> :production
+      "api.firez.one" -> :staging
+      _ -> :unknown
+    end
+
+  sentry_dsn =
+    case api_external_url_host do
+      "api.firezone.dev" ->
+        "https://29f4ab7c6c473c17bc01f8aeffb0ac16@o4507971108339712.ingest.us.sentry.io/4508756715569152"
+
+      "api.firez.one" ->
+        "https://29f4ab7c6c473c17bc01f8aeffb0ac16@o4507971108339712.ingest.us.sentry.io/4508756715569152"
+
+      _ ->
+        nil
+    end
```

Comment:
I might be missing something, but when I made my other comment on setting this using an ENV variable, I was thinking these would be left out of source code altogether.  If these URLs are publicly available, doesn't that allow anyone to pick them up and send data to our sentry account? (e.g. potentially flood our account with data)

Reply:
Well yes, but this is the case with the clients and website analytics, too. Sentry does have knobs for filtering out unwanted traffic and will block erroneous spikes.

We could leave it out of here, but then we won't be able to dev or test against the Sentry integration which has been helpful with the clients.

It seems like Sentry officially designates the DSN as "safe to make public" in their docs here: https://docs.sentry.io/concepts/key-terms/dsn-explainer/#dsn-utilization

Reply:
If this is the norm, then I'm fine with it.

It still feels strange to me and they do say in that same section that they recommend "...having a way to configure DSN dynamically" for client devices, which I would think by extension would apply to our whole code base since our source is public.

> we won't be able to dev or test against the Sentry integration...
I'm not quite following this one.  Wouldn't we be able to just set this in our dev / test environment vars?

Reply:
Since the portal DSN is quite easy to rotate since it's not shipped, I'll leave it inline here for ease of maintainability.